metagraph/src/annotation/binary_matrix/multi_brwt/clustering.cpp

```diff
@@ -16,7 +16,7 @@ namespace matrix {
 
 using mtg::common::logger;
 
-typedef std::vector<std::vector<uint64_t>> Partition;
+typedef std::vector<std::vector<uint32_t>> Partition;
```

Comment:
Should we just make this `Column` as well?

Reply:
this always needs to be 32, it's a specific to this .cpp file

Reply:
ah, no, I confused it with something else. Yeah, this can be either, RAM is not an issue here. But still specific to this file